timeinterval/timeinterval_test.go

```diff
@@ -659,3 +660,97 @@ func mustLoadLocation(name string) *time.Location {
 	}
 	return loc
 }
+
+func TestIntervener_Mutes(t *testing.T) {
+	// muteIn mutes alerts outside business hours in November, using the +1100 timezone.
+	muteIn := `
+---
+- weekdays:
+    - monday:friday
+  location: Australia/Sydney
+  months:
+    - November
+  times:
+    - start_time: 00:00
+      end_time: 09:00
+    - start_time: 17:00
+      end_time: 24:00
+- weekdays:
```

Comment:
Looking at the docs (https://prometheus.io/docs/alerting/latest/configuration/#time_interval), I expected this to mute on Saturdays and Sundays between 00:00 and 09:00, and between 17:00 and 24:00. Is that correct?